packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/bedrock/invoke-model.ts

```diff
@@ -108,6 +108,24 @@ export interface BedrockInvokeModelProps extends sfn.TaskStateBaseProps {
    */
   readonly output?: BedrockInvokeModelOutputProps;
```

Comment:
Should the addition of the s3InputUri and s3OutputUri params deprecate the input/output params? These params appear to result in the same functionality.

Reply:
changes are keeping in mind the original request where user is not able to specify the s3URI in below format, for which they will need something like these new props `s3InputURI`, let me know if you think otherwise

 Input: this.props.input?.s3Location ? { 
   S3Uri: `$prompt.bucket`, 
 } : undefined, 
 Output: this.props.output?.s3Location ? { 
   S3Uri: `$prompt.key`, 
 } : undefined,

Reply:
Would it be better to add an S3Uri param to BedrockInvokeModelInputProps and BedrockInvokeModelOutputProps.  The input/output params were intended to map directly to the Input and Output fields in Step Functions' Bedrock service integration for InvokeModel:
https://docs.aws.amazon.com/step-functions/latest/dg/connect-bedrock.html#connect-bedrock-custom-apis

Reply:
yeah, that's a good point, i've modified the PR to add these properties under bedrockInvokeModelProps